examples/graphql-yoga/src/server.js

```diff
@@ -26,16 +26,16 @@ const typeDefs = `
 
 const resolvers = {
   Query: {
-    pet: resolver(models.Pet),
-    pets: resolver(models.Pet),
-    user: resolver(models.User),
-    users: resolver(models.User),
+    pet: (...args) => resolver(models.Pet)(...args),
+    pets: (...args) => resolver(models.Pet)(...args),
+    user: (...args) => resolver(models.User)(...args),
+    users: (...args) => resolver(models.User)(...args),
```

Comment:
should not be required, unless `graphql-yoga` changed something i guess.

Reply:
I tested here, it's not working without it. I lost so much time trying to debug it rsrs.

Reply:
Wait, you are saying that it's not necessary in Query context, but it's necessary in: 

```
  User: {
    pets: (...args) => resolver(models.User.Pets)(...args),
  },
  Pet: {
    owner: (...args) => resolver(models.Pet.Owner)(...args),
  },
```
?

Reply:
I never personally used `graphql-yoga` but assuming you use the same version as this example they must have made a breaking change.

However, in any case, `users: (...args) => resolver(models.User)(...args)` should be functionally equivelant to `users: resolver(models.User)`. Unless perhaps graphql-yoga checks the function length of the resolver and does something weird.

Reply:
My tests here, you can see that the batch just work when I use (...args)

![teste1](https://user-images.githubusercontent.com/4183877/60352405-2966db80-999e-11e9-9dff-538d96545ae3.png)
![teste2](https://user-images.githubusercontent.com/4183877/60352413-2bc93580-999e-11e9-8193-782fc279006b.png)

Reply:
Isn't this required because the resolver context is now request based and not server based? Had to do this because you need to set it on each request

Reply:
@intellix maybe, I'll search about it.

Reply:
@intellix No the resolver factory should still look at the context for the request, the resolver factory returns it's own method so with this code pattern you just keep recreating the function

Reply:
@mickhansen I found something here. First, I was messing up because I was using yarn, so I fixed it and made some more tests. 

I was testing here how to reproduce the problem, I discovered that the problem is related to dataloader-sequelize 2.0.0. When I update only the graphql-yoga version to 1.8.0 it works without this (...args), but when I update only dataloader to 2.0.0 it doesn't work and it consoles me this warning (no original function findByPk to wrap).

So maybe this is related to this dataloader-sequelize update to version 2.0.0.

Reply:
FWIW, I'm not using `graphql-yoga` and I did need:
```
resolve: (...args) => resolver(relationshipReference)(...args)
```
rather than
```
resolve: resolver(relationshipReference)
```
which did not work.

Relevant package versions:
```
    "dataloader": "^2.0.0",
    "dataloader-sequelize": "^2.3.1",
    "express-graphql": "^0.12.0",
    "graphql": "^15.5.0",
    "graphql-relay": "^0.6.0", // <--- not sure why this was required, but it was once I added dataloader-sequelize
    "graphql-sequelize": "^9.4.3",
```